xonsh/completers/vox.py

```diff
@@ -0,0 +1,32 @@
+import os
+from xonsh.__amalgam__ import builtins
+from xonsh.platform import scandir
+from xonsh.vox import Vox
+
+DEFAULT_ENV_HOME = os.path.expanduser('~/.virtualenvs')
```

Comment:
That is probably because xonsh ins't started up by import time. You might be able to get away with making this a lazy object http://xon.sh/api/lazyasd.html#xonsh.lazyasd.LazyObject  Otherwise it is probably OK to leave as is or move to the usage site or make into a function.